src/shared/logs-show.c

```diff
@@ -76,7 +76,7 @@ static int print_catalog(FILE *f, sd_journal *j) {
                 return log_oom();
 
         if (colors_enabled())
-                fprintf(f, ANSI_GREY "%s" ANSI_NORMAL " " ANSI_GREEN, prefix);
+                fprintf(f, "%s%s %s%s", ansi_grey(), ANSI_NORMAL, ansi_green(), prefix);
```

Comment:
This is incorrect... `prefix` must be before `ANSI_NORMAL`.

Reply:
Fixed.